src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -227,10 +230,7 @@ public function supportsDenormalization($data, $type, $format = null)
         return \class_exists($type) || (\interface_exists($type, false) && $this->classDiscriminatorResolver && null !== $this->classDiscriminatorResolver->getMappingForClass($type));
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function denormalize($data, $class, $format = null, array $context = array())
+    public function denormalize($data, $class, $format = null, array $context = array(), array &$accumulatingContext = array())
```

Comment:
You mean passing `$context` by reference? I would have to try that, but if memory doesn't fail me, I did try and it didn't work: I'll try again and see :) 
The thing is, wouldn't that qualify as BC break too? I'd argue that such approach could lead to harder to find bugs as the method signatures would differ only by a `&`, where the interface would declare a behaviour and the implementation would allow changes to `$context` to ripple out of the method's scope, so I'd probably like an explicit BC break here instead. Perhaps introducing a solution akin to `ContextAwareDenormalizerInterface` could do(it does specify an extra argument for `supportsDenormalization()` compared to `DenormalizerInterface`)?